### DISCUSSION TITLE
Module not showing up in game

### DISCUSSION THREAD

**Discussion by Snake93a** [Jagusti/fvtt-wfrp4e-gmtoolkit#216] [Q&A] [answered]
Hello,

I just installed your module for my system and it is not showing in game.
The module is in the foundry files as follows: ~~AppData\Local\FoundryVTT\Data\modules\wfrp4e-gm-toolkit
the URL adress is manifested
Versions im using:
Foundry Version 10 Build 291 
System: Warhammer Fantasy Roleplay 4td Edition (6.1.4)

screen from ingame: https://ibb.co/NNXjXk3

Any idea what can be causing that?

**Comment by Jagusti** [comment, marked as answer]
Looks like you're on WFRP4e 6.1.4. The latest version of Toolkit needs 6.4.0*, so update the system and you should be fine. 
(* I do need to update the readme)
